src/app/Components/LoginForm.jsx

```diff
@@ -1,19 +1,26 @@
+import {useState} from 'react';
 import "../styles/loginForm-styles.css"
-
+import axios from 'axios';
+import Cookies from 'js-cookie';
+import { useNavigate } from 'react-router-dom/dist';
 function LoginForm() {
+        const [userHandle,setUserHandle]=useState();
+        const [password,setPassword]=useState();
 
+const navigate=useNavigate();
 
-
-    function submitHandler(event) {
+    async function submitHandler(event) {
```

Comment:
As soon as the user clicks the submit button, there should be some loading or spinning GIF or animation till the user is successfully authenticated and redirected to desired route !